docs/_quickstart/build_action.md

```diff
@@ -9,72 +9,74 @@ redirect_from: /quick-start/
 
 ## Add New Data With an Action
 
-![Zapier Action Demo Animation](https://cdn.zapier.com/storage/photos/e216b6e1eaa49626bd18e3ca72b2a513.gif)
+![Zapier Action Demo Animation](https://cdn.zappy.app/7160a6ce9b11c9427cceb1b7241016cc.gif)
 
 _The input form in Zapier actions let users map fields from trigger and previous action steps to your app_
 
-Zapier actions push or put new data into your API from input fields that users set in their Zaps. Zapier includes *search* actions to find data in an app, and *create* actions to add new items via API calls.
+Zapier actions push new data or update existing data into your API from input fields that users set in their Zaps. Zapier includes *search* actions to find data in an app, and *create* actions to add new items via API calls.
 
-Adding an action to your Zapier integration works much like building a trigger. Only here with create actions, Zapier will `POST` or `PUT` the data users enter in the input fields to your API—and *every* trigger needs an input form to gather data and send to your app.
+Adding an action to your Zapier integration works much like building a trigger. Only here with *create* actions, Zapier will `POST` or `PUT` the data users enter in the input fields to your API—and *every* action needs an input form to gather data and send to your app.
 
-Let’s add an _Action_ to our example GitHub integration so Zapier can send data to GitHub. This tells the Zapier platform how to create the Zap Editor form where users can configure their Zaps and map data to and from the API, then tells it how to make calls to the API endpoint.
+Let’s add an _Action_ to our example GitHub integration so Zapier can send data to GitHub. This tells the Zapier Platform how to create the Zap Editor form where users can configure their Zaps and map data to and from the API, then tells it how to make calls to the API endpoint.
 
 ## 1. Configure Action Settings
 
-![Zapier visual builder action settings](https://cdn.zapier.com/storage/photos/f4fba922b339ce68376e38b751bc79f1.png)
+![Zapier Visual Builder action settings](https://cdn.zappy.app/35d98d82cc88d3dc5451f07f14143e34.png)
 
-_Start by entering the core settings for this action_
+_Each action starts with adding details to a form._
 
-Enter your new action settings, as with your trigger. Select a _Create_ action since this action will create new issues, then include a key, name, noun, and description for an issue.
+Enter your new action settings, as with your trigger. Select a _Create_ action since this action will create new issues; then include a key, name, noun, and description for an issue. The type of action cannot be changed once created, so decide whether the [response type you expect to use](https://platform.zapier.com/docs/faq#what-response-type-does-zapier-expect) for your action better meets a _Search_ or _Create_. 
```

Comment:
```suggestion
Enter your new action settings, as with your trigger. Select a _Create_ action since this action will create new issues; then include a key, name, noun, and description for an issue. The type of action cannot be changed once created, so you will need to decide whether the [response type you expect to use](https://platform.zapier.com/docs/faq#what-response-type-does-zapier-expect) for your action would be better served by a _Search_ or a _Create_. 
```